addons/l10n_in_edi/models/res_company.py

```diff
@@ -1,19 +1,32 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import fields, models, api
 
 
 class ResCompany(models.Model):
     _inherit = "res.company"
 
     l10n_in_edi_username = fields.Char("E-invoice (IN) Username", groups="base.group_system")
     l10n_in_edi_password = fields.Char("E-invoice (IN) Password", groups="base.group_system")
-    l10n_in_edi_token = fields.Char("E-invoice (IN) Token", groups="base.group_system")
+    l10n_in_edi_token = fields.Char("E-invoice (IN) Token", groups="base.group_system", compute="_compute_edi_token_and_edi_token_validity", store=True)
```

Comment:
```suggestion
    l10n_in_edi_token = fields.Char("E-invoice (IN) Token",
        groups="base.group_system",
        compute="_compute_edi_token_and_edi_token_validity",
        store=True
    )
```
Readability crosses 99 char